bus/bus.c

```diff
@@ -153,11 +153,11 @@ static void gip_bus_remove(struct device *dev)
 }
 
 #if LINUX_VERSION_CODE < KERNEL_VERSION(5, 15, 0)
-static int gip_bus_remove_compat(struct device *dev)
+static void gip_bus_remove_compat(struct device *dev)
 {
 	gip_bus_remove(dev);
 
-	return 0;
+	return;
```

Comment:
If you don't feel too strongly about it, I would prefer just to drop the "return".

Reply:
I honestly figured it didn't neeed to be there at all, I left it in only because if I change stuff that's not mine I try and stick as close to the original author's intent, as possible.

I will need to remove it and just double-check it still builds but I'm totally cool with just removing it if the build is good to go. Give me 20 to sort this.

Reply:
Sorry that took so long, I have like a zillion tabs open and couldn't;t find this one hahaha.

Builds fine, the commit is there with the change now.

Reply:
Sorry I'm not sure if I'm meant to resolve this conversation, or you. I figured it should be you since I should wait until you are satisfied.... but sorry if I'm doing it wrong!